src/main/webapp/app/modeling-assessment-editor/modeling-assessment-conflict/modeling-assessment-conflict.component.html

```diff
@@ -1,40 +1,60 @@
 <div class="conflict-container">
-    <h2 class="navigation-container">
-        Conflicts
-        <jhi-alert></jhi-alert>
-        <div class="control-container row-container">
+    <h3 class="top-container">
+        <div class="navigation-container row-container">
+            <span jhiTranslate="modelingAssessmentEditor.conflicts">Conflicts</span>
             <button class="btn btn-primary" (click)="onPrevConflict()">
                 <fa-icon icon="arrow-left"></fa-icon>
             </button>
-            <span>{{ conflictIndex + 1 }}/{{ conflicts.length }}</span>
+            <span *ngIf="conflicts">{{ conflictIndex + 1 }}/{{ conflicts.length }}</span>
```

Comment:
I think it also needs to check if conflicts.length === 0, because otherwise it would show 1/0.

Reply:
Good point there. The complete component only makes sense if there are conflicts to display so I added an check in the onInit hook to throw an error when an empty conflict list gets  handed over